openassessment/templates/legacy/edit/oa_edit_peer_assessment.html

```diff
@@ -54,6 +54,18 @@
                         </p>
                     {% endif %}
                 </li>
+                {% if enable_peer_configurable_grading %}
```

Comment:
Should we allow the grading strategy to be changed after the grading has already been calculated? I don’t know if this could generate some kind of inconsistency.

Reply:
Thank you. You raised a valid concern. From an implementation point of view, the calculations happen automatically for each criterion but not for the final grade, so that would be confusing for end-users. What I think we can do is avoiding changing the grading strategy after the problem is released.

Reply:
Done! Thank you again